examples/region_of_interest/SpanROI_interactive_sum.py

```diff
@@ -0,0 +1,48 @@
+"""
+Interactive integration of one dimensional signal
+=================================================
+
+This example shows how to integrate a signal using a interactive ROI
+
+"""
+
+import hyperspy.api as hs
+
+#%%
+# Create a signal:
+s = hs.datasets.artificial_data.get_core_loss_eels_line_scan_signal(True, True)
+
+#%%
+# Create SpanROI
+roi = hs.roi.SpanROI(left=675, right=730)
+
+#%%
+# Slice signal with roi with the ROI. By using the `interactive` function, the
+# output signal ``s_roi`` will update automatically
+# The ROI will be added automatically on the signal figure
+s.plot()
+sliced_signal = roi.interactive(s, axes=s.axes_manager.signal_axes)
+
+#%%
+# Create a placeholder signal for the intregated signal and set metadata
+integrated_sliced_signal = sliced_signal.sum(axis=-1).T
+integrated_sliced_signal.metadata.General.title = "Integrated intensity"
+
+#%%
+# Create the interactive computation, which will update when the ROI ``roi`` is
+# changed, we use the ``out`` argument to place the results of the integration
+# in the placeholder signal defined in the previous step
+hs.interactive(
+    sliced_signal.sum,
+    axis=sliced_signal.axes_manager.signal_axes,
+    event=roi.events.changed,
+    recompute_out_event=None,
+    out=integrated_sliced_signal,
+)
+
+#%%
+# Plot the integrated sum signal
+integrated_sliced_signal.plot()
+
+#%%
+# sphinx_gallery_thumbnail_number = 2
```

Comment:
What is this line needed for? It distracts a bit on the rendered documentation page.

Reply:
Is it just a paste error? Also found in https://hyperspy--3221.org.readthedocs.build/en/3221/auto_examples/model_fitting/plot_residual.html

Reply:
This is to select the matplotlib figure to display as thumbnail in the gallery, otherwise it takes the first one (if I remember correctly), which is the navigator and not a good thumbnail.

Reply:
Indeed, though it is usually set as code-style comment and not in the marked up text:
https://sphinx-gallery.github.io/stable/auto_examples/plot_4_choose_thumbnail.html

I propose to add the comment directly after the plot command with an additional explanatory line as in the suggested changes.